server/sql/dev/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
```

Comment:
how could i run this docker container. I have zero knowledge about docker? what's the connectionstring? is it possible to add the "docker tools" as devDedenpency and create a npm-script that starts the docker container?

Reply:
I only added this to be able to use the mongo db inside a docker container. Because I didn't want to install mongo db locally. So you don't really need to use this if you don't want. But if you want to try it I can add some notes to the readme

Reply:
yes, i'm interested. please add some notes. thanks.